pkg/testing/http_mock.go

```diff
@@ -18,16 +18,21 @@ package testing
 
 import (
 	"bufio"
+	"fmt"
 	"io/ioutil"
 	"net"
 	"net/http"
 	"os"
-	"path/filepath"
 	"regexp"
-	"runtime"
+	"strings"
 	"time"
 )
 
+const (
+	OCTET_STREAM = "application/octet-stream"
+	JSON         = "application/json; charset=utf-8"
```

Comment:
Can be rename `OCTET_STREAM`/`JSON` to `OCTET_STREAM_CONTENT_TYPE` / `JSON_CONTENT_TYPE`.

This feels too generic too me.